packages/components/organisms/f-cookie-banner/f-cookie-banner-static/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@justeat/f-cookie-banner-static",
+  "description": "Fozzie Cookie Banner compiled via vue cli prerender to vanilla js/css",
+  "license": "Apache-2.0",
+  "version": "0.1.0",
```

Comment:
Rather than have a separate version for the static build, can it not just be released as part of the base component (so just a separate build step?)

Reply:
So perhaps could get built to something like `dist/static` in the root of the `f-cookie-banner` package

Reply:
This is exactly what I was trying to achieve. However, I have not been able to chain the static build following the main build, it fails. I can run `yarn build:static-files` from the root but changing the root scripts to run `yarn build && yarn  build:static-files` or using `postbuild: yarn buid:static-files`. Maybe we pair to have a look and see if we can achieve this.

Reply:
Yeh, I'm pretty free this afternoon if you want to pair then?

Reply:
Managed  to get a useful error. vue-cli was linting the compiled output when running from root. I have updated it so the static files are generated as part of the root publish command. Files added to `dist/static`